packages/babel-preset-env/test/index.skip-bundled.js

```diff
@@ -372,4 +372,21 @@ describe("babel-preset-env", () => {
       },
     );
   });
+
+  it("should add .browserslistrc to external dependencies when configPath is specified", () => {
+    const browserslistConfigFile = require.resolve(
+      "./regressions/.browserslistrc",
+    );
+    const { externalDependencies } = babel.transformSync("", {
+      configFile: false,
+      presets: [
+        [babelPresetEnv.default, { configPath: browserslistConfigFile }],
+      ],
+    });
+    expect(externalDependencies).toContain(browserslistConfigFile);
+  });
+
+  it.todo(
+    "should add .browserslistrc to external dependencies when browserslistConfigFile is specified",
+  );
```

Comment:
This case is not yet supported because babel-core resolves the browserslist config in `loadPartialConfig` phase, prior to the `loadFullConfig` phase where `externalDependencies` are prepared.